tests/integration/test_tenants.py

```diff
@@ -30,12 +32,30 @@ def test_tenants_POST(client, auth_headers):
         "staffIDs": [1, 2]
     }
 
+    newTenantWithLease = {
+        "firstName": "Finn",
+        "lastName": "The Human",
+        "phone": "123-555-4321",
+        "propertyID": 2,
+        "occupants": 3,
+        "dateTimeEnd": Time.one_year_from_now_iso(),
+        "dateTimeStart": Time.yesterday_iso(),
```

Comment:
This test is failing, and I think it is because we are passing an datetime objects rather than a stringified iso date. Can you confirm?

Reply:
Well, that is interesting. I'm going to create a test or two and push that up so that I can get a better sense of where it is failing.

Reply:
OK it's passing. This was the problem:
```
    @staticmethod
    def one_year_from_now_iso():
        return datetime.isoformat(datetime.today() + relativedelta(year=1))
```
The `year` parameter should be pluralized; otherwise it seems to return the year 1 instead of the relative year. Still unclear why it didn't fail locally, but on the CI the json response for the failing request was a validation error stating that it was an invalid datetime format.

Feel free to check things over and update if need be. I will give this another once over tomorrow/today, and will most likely get this merged in as I think it is good to go.